examples/How_to_count_tokens_with_tiktoken.ipynb

```diff
@@ -433,7 +434,84 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Tokens: 1445. Reference https://platform.openai.com/docs/guides/vision/calculating-costs\n"
+     ]
+    }
+   ],
+   "source": [
+    "import math\n",
```

Comment:
Could be nice to add a text cell above explaining what's happening here and citing the docs.